### PR TITLE
Frontend wizard restore optimization

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: frontends/web/src/routes/device/setup/components/tampered.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export function Tampered({
+    style = null
+}) {
+    return (
+        <Message type="warning" style={style}>
+            <Alert />
+            {i18n.t('deviceTampered')}
+        </Message>
+    );
+}
```
Comment (KasparEtter) [inline frontends/web/src/routes/device/setup/components/tampered.jsx]: I thought the deal was that new code is written in TypeScript! 😜

**Comment by benma** [comment]
Seems to be based on old commits, e.g. before i18n/en.json moved to locales/en/app.json, and much more.

Please rebase first :)

File: frontends/web/src/routes/device/setup/components/tampered.jsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/routes/device/setup/components/tampered.jsx]: no sorry and I am going to do another change before TS

**Comment by thisconnect** [comment]
rebased

File: frontends/web/src/locales/en/app.json [hunk 2]
```diff
@@ -588,11 +589,16 @@
   },
   "seedRestore": {
     "info": {
-      "button": "Continue",
-      "description": "To continue please insert the micro SD card in to the BitBox.",
-      "title": "How to restore a wallet from a backup"
+      "title": "Restore a wallet",
+      "description_1": "Insert the micro SD card in to the BitBox and click “Continue”",
```
Comment (benma) [inline frontends/web/src/locales/en/app.json]: Please don't use `_` in keys, unless it is used as dynamic context. e.g. `t('my.key', { context: 'foo' })` resolves to `my.key_foo`.

Comment (thisconnect) [reply frontends/web/src/locales/en/app.json]: tried to fix, but some are coming from the backend. i.e. `unlock.error.e109_normal`

Comment (benma) [reply frontends/web/src/locales/en/app.json]: Yes, that is a valid use of context, see [here](https://github.com/digitalbitbox/bitbox-wallet-app/blob/c843d712207594a567f785ba083f2980fc5a6522/frontends/web/src/routes/device/unlock.jsx#L127). Thanks for doing the rest :)

**Comment by benma** [comment]
(this is technically already merged, GH having trouble right now with consistency)

**Pull request merged**